common/src/main/java/journeymap/api/v2/client/display/Overlay.java

```diff
@@ -153,7 +153,7 @@ public Overlay setLabel(@Nullable String label)
     }
 
     /**
-     * The minimum zoom level (0 is lowest) where the polygon should be visible.
+     * The minimum zoom level (UIState.FULLSCREEN_ZOOM_MIN is lowest) where the overlay should be visible.
```

Comment:
Is it okay to mention the constant or should I mention the value directly?

Reply:
that is fine